app/controllers/application_controller.rb

```diff
@@ -3,6 +3,8 @@ class ApplicationController < ActionController::Base
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
 
+  include PublicActivity::StoreController
```

Comment:
Conventionally you'll often find `include`s are placed immediately below the `class` definition, on line 2